### PR TITLE
Fix circular dep between logger.ts and db.ts

### PR DESCRIPTION
**Pull request by rizwanmustafa**

### CODE REVIEW

File: backend/utils/db-logger.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+interface Log {
+  type?: string;
+  timestamp: number;
+  uid: string;
+  event: string;
+  message: string;
+}
+
+const logToDb = async (
```
Comment (Bruception) [inline backend/utils/db-logger.ts:12]: Why is this necessary?

**Comment by rizwanmustafa** [comment]
@Ferotiq This is a backend issue.

File: backend/utils/db-logger.ts (hunk 1 above)
Comment (rizwanmustafa) [reply backend/utils/db-logger.ts:12]: There is a circular dependency between db.ts and logger.ts and I don't think we want that.

Comment (Bruception) [reply backend/utils/db-logger.ts:12]: > There is a circular dependency between db.ts and logger.ts and I don't think we want that.

Is it causing any problems? If not I don’t think this ur necessary.

Comment (rizwanmustafa) [reply backend/utils/db-logger.ts:12]: I guess you are right.
